dvc/command/diff.py

```diff
@@ -111,14 +111,13 @@ def _show_diff(diff, hide_missing=False):
         if not sum(summary.values()):
             return None
 
-        fmt = (
-            "files summary: {added} added, {deleted} deleted,"
-            " {renamed} renamed, {modified} modified"
-        )
-        if not hide_missing:
-            fmt += ", {not in cache} not in cache"
+        states_values = []
+        for state in states:
+            state_value = summary[state]
+            if state_value > 0:
+                states_values.append(f"{state_value} {state}")
 
-        ui.write(fmt.format_map(summary))
+        ui.write("files summary: " + ", ".join(states_values))
```

Comment:
I personally find loops to be confusing for small stuff. Generators/Comprehensions might be clearer to the intent, eg:
```python
summary = ", ".join(generator)
```

Also, `ui.write` is basically a small wrapper around `print`, so you could get rid of "+" and just do:
```python
ui.write("files summary:", summary)
```

WDYT?

Reply:
@skshetry  Fixed :)